clients/discord-bot/src/commands/administrator-commands/misc-commands.marshal.ts

```diff
@@ -54,35 +58,39 @@ export class MiscCommandsMarshal extends Marshal {
     async embed(m: Message, c: MarshalCommandContext): Promise<void> {
         const orgId = c.args.orgId as number;
 
-        const embed = await this.embedService.embed({
-            title: "My Embed Title",
-            description: "My Embed DescriptionLorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sodales ex et nibh volutpat, vel mattis magna feugiat.",
-            fields: [
-                {
-                    name: "Field 1",
-                    value: "Field 1 content",
-                },
-                {
-                    name: "Field 2",
-                    value: "Field 2 content",
-                },
-                {
-                    name: "Field 3",
-                    value: "Field 3 content",
-                },
-                {
-                    name: "Field 4",
-                    value: "Field 4 content",
-                    inline: true,
-                },
-                {
-                    name: "Field 5",
-                    value: "Field 5 content",
-                    inline: true,
-                },
-            ],
-        }, orgId);
+        const embed = await this.embedService.embed(
+            {
```

Comment:
Not a fan of having the `{` on a new line here; @gankoji @bwpikaard @zachstence what are your preferences here?

Reply:
Generally I wouldn't like it, but its like this because on line 91 there is a second parameter being passed to the `embed()` function. Makes sense in this case